templates/aws/cluster_nodes/terraform/pools/outputs.tf

```diff
@@ -19,19 +19,25 @@ output "corral_node_pools" {
     bastion = [for instance in [aws_instance.server[0]] : {
       name = instance.tags.Name // unique name of node
       user = "root" // ssh username
+      ssh_user = var.aws_ssh_user
```

Comment:
I had a similar change pushed back by @igomez06 could you check with him?

Reply:
I think this will be fine Izaac, this a new var.